AOAICustomSkills/function_app.py

```diff
@@ -104,6 +105,34 @@ def call_chat_completion_model(request_body: dict, scenario: str):
         "content": [user_prompt_content]
         }
     ]
+    elif scenario == IMAGE_CAPTIONING_HEADER:
+        logging.info("calling into the image captioning capability")
+        image_base64encoded = request_body.get("data", {}).get("image", "")
+        messages = [ {
+            "role": "system",
+            "content": 
+            [
+                {
+                    "type": "text",
+                    "text": "You are a useful AI assistant who is an expert in machines. You will identify all the parts for the machines in the image sent to you and will formulate the response as a list in JSON."

```

Comment:
this can (and should) be tweaked by our customers when they are using their own Azure functions for whatever specific purposes they have regarding image captioning.

Reply:
Exactly, so we should make it easier, maybe by pulling it from a text file.

Reply:
yup... I will follow up in a subsequent PR to use this pattern across all of our scenarios here. We'll pull content from a JSON file where the user can configure the default system prompt and have multiple options for the chat-suggestion.